services/web/server/src/simcore_service_webserver/login/_2fa.py

```diff
@@ -108,14 +126,18 @@ def _sender():
             f"{message=}",
         )
 
-    await asyncio.get_event_loop().run_in_executor(None, _sender)
+    await asyncio.get_event_loop().run_in_executor(executor=None, func=_sender)
 
 
 #
 # EMAIL
 #
 
 
+class EmailError(RuntimeError):
+    pass
```

Comment:
would it make sense to have an ```errors.py``` module in the login folder?

Reply:
Will do in the next PR where I will actulally be using these errors.